client/pages/private.jsx

```diff
@@ -0,0 +1,7 @@
+import React from "react";
+
+const PrivatePage = () => (
```

Comment:
Just a dummy page to see if authorization works... but now it's still not working.